MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -91,45 +274,232 @@
+				351BEBEC1E5BCC63006FE110 /* StyleButton.swift */,
+				351BEBED1E5BCC63006FE110 /* StyleKitArrows.swift */,
+				351BEBEE1E5BCC63006FE110 /* StyleLabel.swift */,
+				351BEC201E5BD4DC006FE110 /* Supporting files */,
+				351BEBEF1E5BCC63006FE110 /* TurnArrowView.swift */,
+				351BEBF01E5BCC63006FE110 /* UIView.swift */,
+			);
+			path = MapboxNavigationUI;
+			sourceTree = "<group>";
+		};
+		351BEC201E5BD4DC006FE110 /* Supporting files */ = {
+			isa = PBXGroup;
+			children = (
+				351BEC211E5BD506006FE110 /* Resources */,
+				351BEBDA1E5BCC28006FE110 /* Info.plist */,
+			);
+			name = "Supporting files";
+			sourceTree = "<group>";
+		};
+		351BEC211E5BD506006FE110 /* Resources */ = {
+			isa = PBXGroup;
+			children = (
+				351BEC281E5BD530006FE110 /* Assets.xcassets */,
+				351BEC261E5BD529006FE110 /* Base.lproj */,
+				351BEC241E5BD522006FE110 /* de.lproj */,
+				351BEC2C1E5BD541006FE110 /* zh-Hans.lproj */,
+				351BEC2A1E5BD538006FE110 /* Shields.plist */,
+				351BEC221E5BD519006FE110 /* Navigation.storyboard */,
+			);
+			name = Resources;
+			sourceTree = "<group>";
+		};
+		352BBC481E5E78D700703DF1 /* SwiftTests */ = {
+			isa = PBXGroup;
+			children = (
+				352BBC491E5E78D700703DF1 /* Example_SwiftTests.swift */,
+				352BBC4B1E5E78D700703DF1 /* Info.plist */,
+			);
+			name = SwiftTests;
+			path = Examples/SwiftTests;
+			sourceTree = "<group>";
+		};
+		352BBC561E5E78EA00703DF1 /* Objective-CTests */ = {
+			isa = PBXGroup;
+			children = (
+				352BBC571E5E78EA00703DF1 /* Example_Objective_CTests.m */,
+				352BBC591E5E78EA00703DF1 /* Info.plist */,
+			);
+			name = "Objective-CTests";
+			path = "Examples/Objective-CTests";
+			sourceTree = "<group>";
+		};
+		358D14641E5E3B7700ADE590 /* Swift */ = {
+			isa = PBXGroup;
+			children = (
+				358D14651E5E3B7700ADE590 /* AppDelegate.swift */,
+				358D14671E5E3B7700ADE590 /* ViewController.swift */,
+				358D14691E5E3B7700ADE590 /* Main.storyboard */,
+				358D146C1E5E3B7700ADE590 /* Assets.xcassets */,
+				358D146E1E5E3B7700ADE590 /* LaunchScreen.storyboard */,
+				358D14711E5E3B7700ADE590 /* Info.plist */,
+			);
+			name = Swift;
+			path = "Examples/Example-Swift";
+			sourceTree = "<group>";
+		};
+		358D14A71E5E3FDC00ADE590 /* Objective-C */ = {
+			isa = PBXGroup;
+			children = (
+				35C6A3461E5E418D0004CA57 /* AppDelegate.h */,
+				35C6A3471E5E418D0004CA57 /* AppDelegate.m */,
+				35B711E21E5E7E85001EDA8D /* LaunchScreen.storyboard */,
+				35B711E41E5E7E85001EDA8D /* Main.storyboard */,
+				35C6A3481E5E418D0004CA57 /* Assets.xcassets */,
+				35C6A3491E5E418D0004CA57 /* Base.lproj */,
+				35C6A34A1E5E418D0004CA57 /* Info.plist */,
+				35C6A34B1E5E418D0004CA57 /* main.m */,
+				35C6A34C1E5E418D0004CA57 /* ViewController.h */,
+				35C6A34D1E5E418D0004CA57 /* ViewController.m */,
+				358D14A81E5E3FDC00ADE590 /* Supporting Files */,
+			);
+			name = "Objective-C";
+			path = "Examples/Example-Objective-C";
```

Comment:
Yea, sharing storyboards doesn't work. We'd want to verify that it works both programmatically and through storyboard so let's make the Swift example use the storyboard approach and ObjC will init NavUI programmatically.

The main interface is specified in the plist so I'm not sure we can even share that.